### ISSUE TITLE
Add a toggle to disable tour auto-advance

### ISSUE THREAD

**Issue by wolfmanstout** [OneZoom/OZtree#881]
I would recommend adding a toggle to disable auto-advance for tours. I often find myself reading more about each node in other windows, so I want full control over when they advance. This is also an accessibility best practice: https://www.w3.org/TR/WCAG21/#enough-time

**Comment by wolfmanstout** [comment]
I think you could do without auto advance entirely. It's unlikely to precisely match any user's reading speed, so in the best case it is too slow, and I think most people would prefer to click instead of waiting a several seconds. 

If the main driver of this is a museum display, then you could have a kiosk mode which looks for a lack of user interaction and resets the experience. That will actually work better without auto advance anyways, because we can expect more frequent user interaction.

**Comment by lentinj** [comment]
> I think you could do without auto advance entirely

There may be some cases it makes sense, I wouldn't want to throw the option away, but I can't think of concrete examples.

> If the main driver of this is a museum display

It's not, that was just me trying to play devils advocate for a reason to keep it. I'm not sure if there is any strong rationale / UX work behind auto-advance being generally used, or if it's something that made for a more interesting tech demo that's since hung about.

> you could have a kiosk mode

Museum display is onezoom-speak for a kiosk mode. It already has a screensaver / attract mode (which itself is a tour) for this sort of purpose. I'm not sure if the screensaver will cancel an existing tour when activating, or if the active tour will defeat it.

But as you say, this should be fixed rather than used as an excuse for auto-advance.
